docker-compose.yml

```diff
@@ -101,6 +101,8 @@ services:
     profiles: ["default", "ci"]
     ports:
       - 5432:5432
+    expose:
+      - 5432
```

Comment:
Why?

Reply:
btcpay requires a postgres db, so i exposed this one to the host